### DISCUSSION TITLE
Is it possible to add multiple rtmps brodcast?

### DISCUSSION THREAD

**Discussion by abedi-h** [mynaparrot/plugNmeet-server#32] [Q&A] [answered]
Hi

I just viewed the Demo version in your website and i should say congratulations! it is awesome. I tested openmeetings and bigbluebutton in the past but your software seems a lot better. 

I had a question. Is it possible to broadcast to multiple platforms at the same time? we want to stream from plugnmeet to other platforms like telegram, youtube, etc at the same time.

Bests,

**Comment by jibon57** [comment, marked as answer]
Hi,

Thank you. Actually you can do that but you'll have to customize the default behaviour of [plugNmeet-recorder](https://github.com/mynaparrot/plugNmeet-recorder). In this case you'll require to run multiple FFMPEG process to deliver data to different RTMP end points. This is a CPU intensive task. There have different other service providers like Amazon or Vimeo, who are offering retransmission service.

Thanks
